Libs/Common/Shapeworks.h

```diff
@@ -1,5 +1,8 @@
 #pragma once
 
+#include "Libs/Alignment/Transforms/itkThinPlateSplineKernelTransform2.h"
```

Comment:
Not sure if it's necessary, but I really like the fully paths being used for these includes since they are quite confusingly named and likely to be mistaken for actual itk includes. 👍

Reply:
Is there any way to do this without adding these includes to Shapeworks.h?  These will now get included almost everywhere and impact compilation speed of each file.

Reply:
I moved it to ReconstructSurface.h